google/cloud/logging_v2/handlers/_helpers.py

```diff
@@ -211,3 +235,37 @@ def get_request_data():
             return http_request, trace_id, span_id, trace_sampled
 
     return None, None, None, False
+
+
+def get_request_and_trace_data():
+    """Helper to get http_request and trace data from supported web
+    frameworks (currently supported: Flask and Django), as well as OpenTelemetry. Attempts
+    to parse trace/spanID from OpenTelemetry first, before going to Traceparent then XCTC.
```

Comment:
nit this is a bit misleading
```suggestion
    to retrieve trace/spanID from OpenTelemetry first, before going to Traceparent then XCTC.
```